### DISCUSSION TITLE
Show instance data of fx-inspector on open

### DISCUSSION THREAD

**Discussion by rkottmann** [Jinntec/Fore#277] [Ideas]
As a fore user (and debugger) I want to also immediatly see the data in <fx-inspector open></fx-inspector>. 

Currently, the instance data section is collapsed not <detail open="">. It is a bit annoying to always have to click to see details. Especially, when livelreloading a page using fore.

**Comment by JoernT** [comment]
`<fx-inspector open>` should do the trick. It's just an HTML details element used within that element.

**Comment by JoernT** [comment]
otherwise - if you are referring to fx-devtools - these really need an overhaul as its not really up-to-date with latest changes therefore showing some errors in the action section that actually are no errors.
